.github/workflows/build.yml

```diff
@@ -32,4 +32,33 @@ jobs:
 
       - name: Check Docs
         run: |
-          bart check content/* && bart check content/**/*
\ No newline at end of file
+          bart check content/* && bart check content/**/*
+
+      - name: Install spin
+        uses: engineerd/configurator@v0.0.8
+        with:
+          name: "spin"
+          url: "https://github.com/fermyon/spin/releases/download/v0.6.0/spin-v0.6.0-linux-amd64.tar.gz"
+          pathInArchive: "spin"
+
+      - name: Check broken links
```

Comment:
Could we add this test to `package.json` as well? Potentially run the checks using `npm test` or `npm run test`. I think it would be a good idea to be able to test locally before pushing it to GitHub.

Reply:
as the broken-link-checker command line is long (due to the exclusion url's we have), do you think it might be good time to add either a script that is triggered by npm test (.build/check-broken-link.sh)  OR may be a Makefile?

Reply:
Since we already have scripts to generate the search index, we could go down the script route but I do not have a strong preference though.

Reply:
ok, i have added the link checker script to `.build` folder.